apex/pyprof/README.md

```diff
@@ -242,7 +259,7 @@ The above steps should result in a permanent change.
 
 _Temporary solution_
 
-When running on bare metal, you can run nvprof with `sudo`.
+When running on bare metal, you can run nsys with `sudo`.
```

Comment:
Please change it to "... can run nvprof or nsys with `sudo`".

Reply:
Updated